modules/nuxt-easy-redirect.yml

```
@@ -0,0 +1,17 @@
+name: nuxt-easy-redirect
+description: Redirects made easy for Nuxt 3
```

```suggestion
description: Redirects made easy for Nuxt
```
Please do not advertize with a specific Nuxt version, see:
https://nuxt.com/docs/guide/going-further/modules#do-not-advertize-with-a-specific-nuxt-version